ui/src/components/Orchestration/RpcEndpoints.tsx

```diff
@@ -0,0 +1,6 @@
+const RpcEndpoints = {
+osmosis: 'http://127.0.0.1:26655', 
+agoric: 'http://127.0.0.1:26657', 
+};
```

Comment:
this doesn't look like it's indented in the conventional style. do we have `yarn lint` set up in this repo?

Reply:
did you say you got `yarn lint` working? I'm struggling to get it to work:

```
10:13 connolly@bldbox$ yarn lint
Unknown Syntax Error: Command not found; did you mean one of:
```

version I have checked out is:

 - 2024-08-13 15:31 919cf28 create buttons are now a group for both handleCreateAccount and handleCreateAndFund

Reply:
hm, that commit should work for you. what directory are you running the `yarn lint` from? @dckc 

You can run this from TLD:
```
yarn workspace dapp-agoric-orca-contract lint
or simply 
make lint
```

or from `/contract`, you can just run `yarn lint`

result

<details>

```console
ontract % yarn workspace dapp-agoric-orca-contract lint

/Users/jovonni/Documents/projects/devtes/dapp-orchestration-basics/contract/src/orca.contract.js
  160:33  warning  The first `await` appearing in an async function must not be nested  @jessie.js/safe-await-separator

/Users/jovonni/Documents/projects/devtes/dapp-orchestration-basics/contract/src/orca.proposal.js
  35:1  warning  Missing JSDoc @param "config" type  jsdoc/require-param-type

/Users/jovonni/Documents/projects/devtes/dapp-orchestration-basics/contract/src/platform-goals/start-contract.js
  39:1  warning  Missing JSDoc @param "privateArgs" type  jsdoc/require-param-type

✖ 3 problems (0 errors, 3 warnings)

```

</details>

Reply:
re lint, see https://github.com/Agoric/dapp-orchestration-basics/pull/14

Reply:
`yarn lint` from tld throws:

```console
dapp-orchestration-basics % yarn lint
Unknown Syntax Error: Command not found; did you mean one of:

  0. yarn workspaces list [--since] [-R,--recursive] [--no-private] [-v,--verbose] [--json]
  1. yarn workspaces focus [--json] [--production] [-A,--all] ...
  2. yarn workspaces foreach [--from #0] [-A,--all] [-R,--recursive] [-W,--worktree] [-v,--verbose] [-p,--parallel] [-i,--interlaced] [-j,--jobs #0] [-t,--topological] [--topological-dev] [--include #0] [--exclude #0] [--no-private] [--since] [-n,--dry-run] <commandName> ...

While running workspaces run lint
jovonni@jovonnis-MBP dapp-orchestration-basics % 


```